pkg/col/coldata/BUILD.bazel

```diff
@@ -12,12 +12,14 @@ go_library(
         "nulls.go",
         "testutils.go",
         "vec.go",
+        "vec_tmpl.go",
```

Comment:
@yuzefovich any idea why this file wasn't her and/or why `dev gen bazel` suddenly decided to add it now?

Reply:
I reverted the changes to the coldata/colexec stuff for now.